HW6/task3.py

```diff
@@ -0,0 +1,8 @@
+# Write a function is_prime that takes 1 argument - a number from 2 to 1000, and returns True if it is a prime number, and False otherwise.
+
+def is_prime(number: int):
+    if number in range(2, 1001):
+        return True
+    return False
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/218327266-f4b0402f-f99d-43c5-9c64-fc4b1a3bf255.png)
4 is not a prime number. Please read about prime numbers